### DISCUSSION TITLE
Total unique visitors is not equal to the sum of unique visitors per day

### DISCUSSION THREAD

**Discussion by nfbarreto** [plausible/analytics#2974] [Anything Else]
Hi,

I was doing some tests with Plausible and noticed that the aggregated unique visitors metric on top of the dashboard doesn't equal the sum of unique users per day.  I did this in the demo dashboard for the month of April 2023.

![image](https://github.com/plausible/analytics/assets/32324161/c9871226-0654-4a52-b58f-124788aff4e7)

![image](https://github.com/plausible/analytics/assets/32324161/3cfe7998-2b5e-4d61-b247-65e076a72873)

In the documentation, I saw that distinct users only exist on a daily basis. 

Can someone please tell me where this difference is coming from? 126687 (aggregated metric for the entire month) vs 140075 (summing April daily users). 

Thank you!

**Comment by metmarkosaric** [comment]
Hi @nfbarreto! This is expected. The difference comes from the sessions that are active during the daily reset. So if I visit the site 5 minutes before the reset at midnight and my session continues until 5 minutes past the reset, my session will count as one unique visitor on each of the two days while it will count only as 1 unique visitor in the monthly view.

**Reply by nfbarreto** [reply]
Hi @metmarkosaric 

I thought the user ID was generated for every hit. But if it was the case, for the example you shared above, we would have a single session with two different users.

Thank you for the explanation.
